notebooks/how_to_pyscf.md

```diff
@@ -0,0 +1,117 @@
+# How to write PySCF output into TREXIO format
+
+This tutorial shows how to use PySCF (https://pyscf.org/) to save computational results in TREXIO format. 
+
+## Importing TREXIO
+
+TREXIO is avialble throught the pyscf-forge repository (https://github.com/pyscf/pyscf-forge). After installing PySCF-FORGE, you can access the TREXIO library within PySCF using the following import statement:
+
+```python
+from pyscf.tools import trexio
+```
+
+## How to register RHF results
+
+Once a molecule is defined, you can register the Restricted Hartree-Fock (RHF) results into TREXIO. If needed, it is also possible to register UHF wavefunction.
+In this tutorial, the HDF5 backend is used to store the results.
+
+```python
+# Perform RHF calculation
+mf = scf.RHF(mol)
+mf.kernel()
+
+# Save results in TREXIO format
+trexio_file = 'rhf.hdf5'
+to_trexio(mf, trexio_file)
+```
+The ```to_trexio``` function saves various data, including the basis set, molecule specifications, and molecular orbitals, into the TREXIO file.
+
+## Storing Two-Electron Integrals
+ 
+If required, you can also register the two-electron integrals in AO basis using the following commands:
+
+```python
+# Compute two-electron integrals
+eri = mol.into('int2e')
+
+# Compute two-electron integrals
+write_eri(eri, trexio_file)
+```
+
+## Saving determinants after a sCI or FCI calculations
+
+You may want to register a determinant list after a CASCI/CASSCF, selected CI (sCI), or full CI (FCI) calculation. To avoid inconsistencies, it is recommended to save this data in a separate TREXIO file from the one used for RHF results.
+
+
+```python
+norb = 10
+nelec = 4
+                          
+# Initialize CASCI calculation with Selected CI as the solver
+mcscf = mcscf.casci_symm.CASCI(mf, norb, nelec)
+mcscf.fcisolver = selected_ci_spin0_symm.SelectedCI(rhf)
+
+#Perform  CASCI calculation
+mcscf.kernel()
+ 
+#Save determinants into TREXIO
+trexio_file = 'my_det.hdf5'
+det_to_trexio(mcscf, norb, nelec, trexio_file)
+```
+
+The ```det_to_trexio``` function registers the number of MO, number of alpha and beta electrions, total number of determinants, coefficients and the determinants in bitfield format.
+
+## Applying a Threshold for Determinants
+You can filter determinants by their coefficients, saving only those that exceed a specified threshold:
```

Comment:
This is not recommended as you might produce an inconsistent wave function data (see Anthony's comment elsewhere). Please remove this section.